turbinia/workers/artifact.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Task for running Plaso."""
+
+from __future__ import unicode_literals
+
+import os
+
+from turbinia import config
+from turbinia.evidence import ExportedFileArtifact
+from turbinia.workers import TurbiniaTask
+
+
+class FileArtifactExtractionTask(TurbiniaTask):
+  """Task to run image_export (log2timeline)."""
+
+  def __init__(self, artifact_name):
+    super(FileArtifactExtractionTask, self).__init__()
+    self.artifact_name = artifact_name
+
+  def run(self, evidence, result):
+    """Task that process data with Plaso.
```

Comment:
Maybe something slightly more descriptive like "Task that extracts artifacts with Plaso."?  Or even just s/process/processes/

Reply:
Done